cmd/minikube/cmd/config/config.go

```diff
@@ -91,6 +91,14 @@ var settings = []Setting{
 		name: "kubernetes-version",
 		set:  SetString,
 	},
+	{
+		name: "dashboard",
```

Comment:
Could we nest these inside another section, instead of having them at the top level?